packages/clay-modal/stories/index.tsx

```diff
@@ -47,17 +47,20 @@ const ModalWithState: React.FunctionComponent<IProps> = ({
 };
 
 const MyApp: React.FunctionComponent<any> = () => {
-	const [, disptach] = useContext(Context);
+	const [, dispatch] = useContext(Context);
 
 	return (
 		<ClayButton
 			displayType="primary"
 			onClick={() =>
-				disptach &&
-				disptach({
+				dispatch({
 					payload: {
 						body: <h1>{'Hello world!'}</h1>,
-						footerLast: <ClayButton>{'Primary'}</ClayButton>,
+						footer: [
```

Comment:
Are you able to do the same thing as above instead of explicit `undefined`? not totally necessary, but since I just learned of this, I can't help but see it lol

```jsx
[,, <ClayButton>{'Primary'}</ClayButton>]
```

Reply:
Yeah it is possible.